include/dsn/tool/latency_tracer.h

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2017-present, Xiaomi, Inc.  All rights reserved.
+// This source code is licensed under the Apache License Version 2.0, which
+// can be found in the LICENSE file in the root directory of this source tree.
+
+#pragma once
+#include <iterator>
+#include <string>
+#include <map>
+#include <dsn/service_api_c.h>
+#include <dsn/dist/fmt_logging.h>
+
+namespace dsn {
+namespace tool {
+
+struct latency_tracer
+{
+public:
+    uint64_t id;
+    std::string name;
+    uint64_t start_time;
+    uint64_t last_time;
```

Comment:
I think you can add some comments to show what's the difference between last_time and end_time